### DISCUSSION TITLE
Sistemi di Congruenze (Metodo Base E Sostituzione)

### DISCUSSION THREAD

**Discussion by Frisbon** [sapienzastudentsnetwork/algebra#381] [Esercizi Step-by-Step]
_Se l'algoritmo non é chiaro (o non é corretto lol) **corregetemi**._

Dato un sistema con equazioni del tipo:
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/84731341/21cec8cb-ad58-4d79-87f0-d357e399f760)

![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/84731341/2a28ee74-de7b-4e12-83e4-0408c7dcf882)

**Comment by daromacarlo** [comment]
https://www.youtube.com/watch?v=V-nbCGldGas
